src/TicketingService.Abstractions/Ticket.cs

```diff
@@ -13,8 +14,17 @@ public enum TicketStatus
 {
     Created,
     Pending,
-    Complete
+    Complete,
+    Faulted
 }
 
 [JsonConverter(typeof(TicketResultJsonConverter))]
 public record TicketResult(object? Result);
+
+public record TicketError(string ErrorMessage, string ErrorCode)
+{
+    public override string ToString()
+    {
+        return JsonSerializer.Serialize(this);
```

Comment:
fyi @ArneD , I've added a custom ToString() implementation because the TicketResultJsonConvert (see above) calls ToString() on TicketResult.Result, see https://github.com/Informatievlaanderen/ticketing-system/blob/main/src/TicketingService.Abstractions/TicketResultJsonConverter.cs#L11

Maybe we should review this?